src/libraries/Microsoft.PowerFx.Core/Public/Config/SymbolTable.cs

```diff
@@ -66,6 +66,11 @@ private DName ValidateName(string name)
             return new DName(name);
         }
 
+        public SymbolTable()
+        {
+            EnumStoreBuilder = new EnumStoreBuilder();
+        }            

```

Comment:
@jas-valgotar - fyi

Reply:
discussed - this is to prevent silently ignoring enums on line 325. 
That's probably a deeper issue to fix.